badge_server/main.py

```diff
@@ -137,24 +138,46 @@ def get_highest_status(cls, statuses: Iterable[enum.Enum]):
 }
 
 
-def _get_supported_versions(package_name: str) -> List[int]:
-    """Gets the given package's supported python versions
+def _get_missing_details(package_names: List[str],
+                         results: Iterable[CompatibilityResult]
+                         ) -> Optional[str]:
+    """Gets the details for any missing data (if there is any)
 
     Args:
-        package_name: The name of the package to look up, e.g. "tensorflow".
+        package_names: A list of length 1 or 2 of the package name(s) to look
+            up, e.g. ["tensorflow"], ["tensorflow", "opencensus"].
+        results: A list of length 1 or 2 of the `CompatibilityResults` for the
+            given package(s). Results should come from datastore.
```

Comment:
I mean specifically that they should be single or pairwise compatibility results i.e. they should be for the same set of packages. You could instead write something like:

"The package names in the CompatibilityResults should match the `package_names` argument"

Reply:
Oh ok got it - done